### PR TITLE
OCD-3693 - Automate Cures Reporting Phase 1 Charts (Charts 4-6)

### PR DESCRIPTION
**Pull request by tmy1313**

### CODE REVIEW

File: dev/openchpl_data-model.sql [hunk 1]
```diff
@@ -3506,6 +3506,48 @@ CREATE UNIQUE INDEX deprecated_api_usage_unique_api_key_and_deprecated_api
...
+      ON UPDATE CASCADE ON DELETE RESTRICT,
+	CONSTRAINT original_criterion_id_fk FOREIGN KEY (original_criterion_id)
+      REFERENCES openchpl.certification_criterion (certification_criterion_id) MATCH FULL
+      ON UPDATE CASCADE ON DELETE RESTRICT,
+	CONSTRAINT cures_criterion_id_fk FOREIGN KEY (cures_criterion_id)
+      REFERENCES openchpl.certification_criterion (certification_criterion_id) MATCH FULL
+      ON UPDATE CASCADE ON DELETE RESTRICT
+);
+
+CREATE TABLE IF NOT EXISTS openchpl.cures_listing_statistics_by_acb (
```
Comment (kekey1) [inline dev/openchpl_data-model.sql]: "IF NOT EXISTS" may not be needed here

File: changes/ocd-3693.sql [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+	statistic_date timestamp NOT NULL DEFAULT NOW(),
+	creation_date timestamp NOT NULL DEFAULT NOW(),
+	last_modified_date timestamp NOT NULL DEFAULT NOW(),
+	last_modified_user bigint NOT NULL,
+	deleted bool NOT NULL DEFAULT false,
+	CONSTRAINT cures_lsiting_statistics_pk PRIMARY KEY (id),
+	CONSTRAINT certification_body_id_fk FOREIGN KEY (certification_body_id)
+      REFERENCES openchpl.certification_body (certification_body_id) MATCH FULL
+      ON UPDATE CASCADE ON DELETE RESTRICT
+);
```
Comment (kekey1) [inline changes/ocd-3693.sql:41]: Do we want the audit functions with these tables? They feel temporary-ish (like for 3 years) so maybe we do not... but wanted to ask.

**Review by kekey1 — CHANGES_REQUESTED** [review]

**Review by andlar — DISMISSED** [review]

File: dev/openchpl_data-model.sql [hunk 3]
```diff
@@ -3508,6 +3508,44 @@ CREATE UNIQUE INDEX deprecated_api_usage_unique_api_key_and_deprecated_api
 ON openchpl.deprecated_api_usage(api_key_id, deprecated_api_id)
 WHERE deleted = false;
 
+<<<<<<< HEAD
```
Comment (andlar) [inline dev/openchpl_data-model.sql]: This doesn't look right

**Review by andlar — DISMISSED** [review]

**Review by kekey1 — DISMISSED** [review]

File: dev/openchpl_data-model.sql [hunk 4]
```diff
@@ -3619,6 +3658,7 @@ CREATE TABLE openchpl.deprecated_response_field_api_usage (
 CREATE UNIQUE INDEX deprecated_response_field_api_usage_unique_record
 ON openchpl.deprecated_response_field_api_usage(api_key_id, deprecated_api_id)
 WHERE deleted = false;
+>>>>>>> staging
```
Comment (andlar) [inline dev/openchpl_data-model.sql]: This shouldn't be here, right?

File: dev/openchpl_data-model.sql (hunk 3 above)
Comment (andlar) [reply dev/openchpl_data-model.sql]: Merge conflict issue?

**Review by andlar — CHANGES_REQUESTED** [review]

**Review by kekey1 — APPROVED** [review]

**Review by andlar — APPROVED** [review]

**Pull request merged**